### ISSUE TITLE
@tabler/icons-react dependency is huge

### ISSUE THREAD

**Issue by DavidZbarsky-at** [jotaijs/jotai-devtools#87]
Hi, thanks for this great project.

Is is possible to rejigger the build process a bit so that installing this dependency doesn't force installation of `@tabler/icons-react`, which is 64MB? I noticed you are only using a few icons, is it possible to make `@tabler/icons-react` a devDependency and distribute a built artifact that only has the icons you need (or something like that)?
Things are better now than when using `@tabler/icons` which was 100MB, but would be great to cut it down even more.

Our project has a big node_modules footprint which hurts our CI times, and while there are many offenders, this package is one of the biggest ones.

Thanks for you consideration!

**Comment by arjunvegda** [comment]
Hi @DavidZbarsky-at 

Thanks for reaching out! I'll be AFK for a week [traveling], so, I'll look deeper into this in a couple of weeks. 

For now, could you test if [this PR](https://github.com/jotaijs/jotai-devtools/pull/88) fixes it? Here are [the instructions](https://github.com/jotaijs/jotai-devtools/pull/88#issuecomment-1645654043) on installing the canary version.

**Comment by DavidZbarsky-at** [comment]
Thanks for taking a look! That certainly looks like the right fix to me. I ran into a bit of trouble testing because we use yarn and attempting to point to the hosted canary package yields this error: 
```
https://pkg.csb.dev/jotaijs/jotai-devtools/commit/b82a6aa5/jotai-devtools isn't supported by any available resolver
``` 
After reading https://github.com/yarnpkg/berry/issues/3866, I think tweaking your uploads so that the file has a `.tgz` extension should solve it. After I downloaded + renamed it, yarn was able to read it and uninstall `@tabler/icons` from our workspace

**Comment by arjunvegda** [comment]
Awesome! Thanks for taking a look! I'll have to do a bit of testing that I cannot get to until after next week. But you're welcome to continue using the Canary version in your app until then 🙌 

> After reading https://github.com/yarnpkg/berry/issues/3866

I don't think we need to do that manually 🤔 

Does the second command (`# yarn 2, 3`) not work?

<img width="857" alt="image" src="https://github.com/jotaijs/jotai-devtools/assets/14841132/b3d162cb-65db-4c0d-8fa9-4c06128fe252">

**Comment by DavidZbarsky-at** [comment]
No, it doesn't work:
```
yarn add jotai-devtools@https://pkg.csb.dev/jotaijs/jotai-devtools/commit/b82a6aa5/jotai-devtools/_pkg.tz
➤ YN0000: ┌ Resolution step
➤ YN0001: │ Error: jotai-devtools@https://pkg.csb.dev/jotaijs/jotai-devtools/commit/b82a6aa5/jotai-devtools/_pkg.tz isn't supported by any available resolver
```

**Comment by arjunvegda** [comment]
> No, it doesn't work:

The extension is incorrect. It should be `.tgz`. I'd copy-paste it (yarn 2, 3) verbatim from [the CodeSandbox page](https://ci.codesandbox.io/status/jotaijs/jotai-devtools/pr/88/builds/400411)

**Comment by arjunvegda** [comment]
Also, the fix has been released as part of `0.6.1` 🚀
